server/rest/src/main/java/org/infinispan/rest/ALPNHandler.java

```diff
@@ -76,22 +78,56 @@ public void configurePipeline(ChannelPipeline pipeline, String protocol) {
     */
    protected void configureHttp2(ChannelPipeline pipeline) {
       pipeline.addLast(getHttp11To2ConnectionHandler());
+      addHttp1Handlers(pipeline);
+   }
+
+   /**
+    * Adds the commons Http1 handlers to the pipeline.
+    */
+   private void addHttp1Handlers(ChannelPipeline pipeline) {
```

Comment:
maybe change the method name to `addCommonHandlers`. it isn't clear for me if this method should register only HTTP1 or HTTP2 or both handlers :)